roles/cloud_connector/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+foreman_cloud_connector_console_redhat_host: cert.cloud.redhat.com
```

Comment:
in which case would one want to override this?

Reply:
I think in the past folks were testing sat+receptor against a testing environment in cloud

Reply:
Good question - I would have to ask if there is a staging environment for testing.

Reply:
fair :)